docs/src/app/components/MobileTearSheet.js

```diff
@@ -37,10 +39,14 @@ class MobileTearSheet extends Component {
         display: 'block',
         position: 'relative',
         marginTop: -10,
-        width: 360,
```

Comment:
Passing down the width property in context wasn't necessary indeed but I would point out the following:
1. having maxWidth: 360 in bottomTear isn't necessary because the root (its container) already has this property
2. I would add width: '100%' in root for a better visualisation on mobile
3. having the display: 'flex' property on the container div of MobileTearSheet (in ExampleSettings.js) does this:

![overflow-settings-list](https://cloud.githubusercontent.com/assets/6530577/16546200/cbb6572c-413a-11e6-89b9-50d1e126fc5f.png)

I'll do a new PR with my final version for this if you want to have a look.